### PR TITLE
chore: v2.0.0 deployments

### PR DESCRIPTION
**Pull request by jaypaik**
- Deployment artifacts for both MultiOwnerLightAccount and LightAccount (and their factories) on 14 networks (7 chains and their testnets).
- Migrated `Deployments.md` to the deployments file structure used in modular-account to standardize.

### CODE REVIEW

File: deployments/mainnets/zora-mainnet.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+| ------- | -------------------------------------------- | ------------------------------------------------------------------------------------------- | -------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
+| v2.0.0  | `0x0000000000400CdFef5E2714E63d8040b700BC24` | [explorer](https://explorer.zora.energy/address/0x0000000000400CdFef5E2714E63d8040b700BC24) | `0x00000000000000000000000000000000000000005f1ffd9d31306e056bcc959b` | [run](./broadcast/Deploy_LightAccountFactory.s.sol/7777777/run-1714101518.json) |
+| v1.1.0  | 0x00004EC70002a32400f8ae005A26081065620D20   | [explorer](https://explorer.zora.energy/address/0x00004EC70002a32400f8ae005A26081065620D20) | `0x4e59b44847b379578588920ca78fbf26c0b4956c3406f3bdc271500000c2f72f` | [run](./broadcast/Deploy_LightAccountFactory.s.sol/7777777/run-1710446999.json) |
```
Comment (adam-alchemy) [inline deployments/mainnets/zora-mainnet.md]: Super small nit: the address column here is missing the code style via "`" chars.

(Side note: this is something an automated deployment generating script would fix).

File: deployments/mainnets/zora-mainnet.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## LightAccount
+
+| Version | Address                                      | Explorer                                                                                    | Run                                                                             |
+| ------- | -------------------------------------------- | ------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
+| v2.0.0  | `0x8E8e658E22B12ada97B402fF0b044D6A325013C7` | [explorer](https://explorer.zora.energy/address/0x8E8e658E22B12ada97B402fF0b044D6A325013C7) | [run](./broadcast/Deploy_LightAccountFactory.s.sol/7777777/run-1714101518.json) |
+| v1.1.0  | 0xae8c656ad28F2B59a196AB61815C16A0AE1c3cba   | [explorer](https://explorer.zora.energy/address/0xae8c656ad28F2B59a196AB61815C16A0AE1c3cba) | [run](./broadcast/Deploy_LightAccountFactory.s.sol/7777777/run-1710446999.json) |
```
Comment (adam-alchemy) [inline deployments/mainnets/zora-mainnet.md]: Also missing code style here

File: deployments/testnets/arb-goerli.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+> [!IMPORTANT]  
+> This network is deprecated and is no longer being deployed to.
```
Comment (adam-alchemy) [inline deployments/testnets/arb-goerli.md:2]: Nice format

**Review by adam-alchemy — APPROVED** [review]

File: deployments/mainnets/zora-mainnet.md (hunk 1 above)
Comment (jaypaik) [reply deployments/mainnets/zora-mainnet.md]: good eye!

**Pull request merged**